script/MangroveDeployer.s.sol

```diff
@@ -20,32 +20,40 @@ contract MangroveDeployer is Deployer {
     innerRun({
       chief: envHas("CHIEF") ? vm.envAddress("CHIEF") : broadcaster(),
       gasprice: envHas("GASPRICE") ? vm.envUint("GASPRICE") : 1,
-      gasmax: envHas("GASMAX") ? vm.envUint("GASMAX") : 2_000_000
+      gasmax: envHas("GASMAX") ? vm.envUint("GASMAX") : 2_000_000,
+      gasbot: envHas("GASBOT") ? vm.envAddress("GASBOT") : fork.get("Gasbot")
     });
     outputDeployment();
   }
 
-  function innerRun(address chief, uint gasprice, uint gasmax) public {
+  function innerRun(address chief, uint gasprice, uint gasmax, address gasbot) public {
     broadcast();
     if (forMultisig) {
-      mgv = new Mangrove{salt:salt}({governance: chief, gasprice: gasprice, gasmax: gasmax});
+      oracle = new MgvOracle{salt: salt}({governance_: chief, initialMutator_: gasbot});
     } else {
-      mgv = new Mangrove({governance: chief, gasprice: gasprice, gasmax: gasmax});
+      oracle = new MgvOracle({governance_: chief, initialMutator_: gasbot});
+    }
+    fork.set("MgvOracle", address(oracle));
+
+    broadcast();
+    if (forMultisig) {
+      mgv = new Mangrove{salt:salt}({governance: broadcaster(), gasprice: gasprice, gasmax: gasmax});
```

Comment:
I see we invoke broadcaster() in other places after broadcast() - the code for broadcaster is not trivial - maybe we should add a comment to it that it does not cause transactions so that this is obvious?